### DISCUSSION TITLE
"Close All Apps" Icon

### DISCUSSION THREAD

**Discussion by Fskum** [elrumo/macOS_Big_Sur_icons_replacements#1222] [Icon request]
I am looking for a special icon. I have created an "Automator" shortcut that can close all open programs with one tap, but I am missing a proper icon. Right now I'm just using the AdBlocks icon.

I would hear if one or more would give a shot, on an icon that illustrates an app that can close all apps in one tap. Only requirement is it should be simple, and fit the other icons without sticking out too much.

Thanks in advance
<img width="500" alt="One Tap" src="https://user-images.githubusercontent.com/86257880/123135257-dbccbe80-d451-11eb-9a5c-bf5b23a1afdd.png">

**Comment by blaichch** [comment]
I think I got carried away a bit too far for your request and I'm not completely happy with it. But at least want to offer it to you. I might give it another try in the future.

<img width="512" alt="Close Windows 2" src="https://user-images.githubusercontent.com/12624897/124566435-54b11a80-de43-11eb-8a77-2a6564fc7fed.png">

**Reply by QuintB** [reply]
This is really cool, thank you for sharing :)

**Reply by Fskum** [reply]
I've not been here in a while, but it's a really nice icon. Thank you very much, well done.

**Comment by johnlamox** [comment]
Did something more minimal.

![Closing Time](https://user-images.githubusercontent.com/57749200/126899751-8bb75742-5279-4a0c-ae20-35284f5a3b47.png)

Will also add it to my uploads as "Closing Time"-Icon. :)

**Reply by Fskum** [reply]
It's really good too, I love the simple expression. Thank you very much for your great work.
